lib/imagemounter/index.js

```diff
@@ -0,0 +1,119 @@
+import { readFile, createReadStream } from 'fs';
+import { BaseServicePlist } from '../base-service';
+import { fs } from '@appium/support';
+import B from 'bluebird';
+import log from '../logger';
+const MOBILE_IMAGE_MOUNTER_SERVICE_NAME = 'com.apple.mobile.mobile_image_mounter';
+const { exists, lstat } = fs;
+class ImageMounter extends BaseServicePlist {
+
+    _checkReturnError(ret) {
+        if (ret.Error) {
+            throw new Error(ret.Error);
+        }
+    }
```

Comment:
if you were going to make it a static method, maybe this:

```suggestion
    /**
     * Asserts the response from a `PlistService` does not contain an error
     * @private
     * @param { {Error?: string} } ret
     * @returns {void}
     */
    static assertReturnError(ret) {
      if (ret.Error) {
        throw new Error(ret.Error);
      }
    }
```